README.md

```diff
@@ -14,6 +14,7 @@ OpenTrials is an app to explore, discover, and submit information on clinical tr
 ### Requirements
 
 * Node 5.8
+  * gulp
```

Comment:
As `gulp` is in our `package.json`'s dependencies, this shouldn't be needed. You could argue that `gulp` shouldn't be in our dependencies, but we need it there for building on Heroku.